app/lib/nomis_client/people.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class NomisClient
+  class People
+    class << self
+      def get(prison_number)
+        attributes_for(
+          NomisClient.get("/prisoners/#{prison_number}").parsed.first
+        )
+      end
+
+      def attributes_for(person)
+        {
+          nomis_prison_number: person['offenderNo'],
+          last_name: person['lastName'],
+          first_names: [person['firstName'], person['middleNames']].compact.join(' '),
```

Comment:
Makes sense to map the keys from camel to snake case but I don't think this kind of mapping stuff belongs in the client library. It's a separate concern.

Reply:
I agree with that, will change it